redash/query_runner/__init__.py

```diff
@@ -104,6 +105,25 @@ def to_dict(cls):
         }
 
 
+class BaseSQLQueryRunner(BaseQueryRunner):
+    def __init__(self, configuration):
+        super(BaseSQLQueryRunner, self).__init__(configuration)
+
+    def get_schema(self):
+        schema_dict = {}
+        self._get_tables(schema_dict)
+        self._get_tables_stats(schema_dict)
+        return schema_dict.values()
+
+    def _get_tables(self, schema_dict):
+        return []
+
+    def _get_tables_stats(self, tables_dict):
+        for t in tables_dict.keys():
+            if type(tables_dict[t]) == dict:
+                res = self._run_query_internal('select count(*) as cnt from %s' % t)
+                tables_dict[t]['size'] = res[0]['cnt']
```

Comment:
Currently we might load the schema the first time as part of a web request, hence it needs to be relatively fast, so this can't be the default implementation, specially not for Postgres (because of Redshift).

We can either change this default implementation, or have it run as a Celery task.